src/patternfly/components/List/list.scss

```diff
@@ -22,6 +40,28 @@
     list-style: var(--pf-c-list--ul--ListStyle);
   }
 
+  .pf-c-list__item {
+    display: flex;
+  }
+
+  .pf-c-list__item-icon {
+    display: flex;
+    min-width: var(--pf-c-list__item-icon--MinWidth);
+    margin-top: var(--pf-c-list__item-icon--MarginTop);
+    margin-right: var(--pf-c-list__item-icon--MarginRight);
+    font-size: var(--pf-c-list__item-icon--FontSize);
+    color: var(--pf-c-list__item-icon--Color);
+  }
+
+  &.pf-m-lg {
+    .pf-c-list__item-icon {
```

Comment:
Custom properties are inherited, so you shouldn't need to define this selector, and just apply it to `.pf-c-list.pf-m-lg`